### DISCUSSION TITLE
在子组件中获取 viewer 实例不成功，显示injection "Symbol(VueCesium)" not found.

### DISCUSSION THREAD

**Discussion by kfcvs** [zouyaoji/vue-cesium#186] [Q&A]
我试图在子组件中通过useVueCesium 函数获取 viewer 实例显示如下错误
![image](https://user-images.githubusercontent.com/31976998/162259552-bd0e8905-3680-4a5d-9565-c1f4ed48a6d4.png)
[按照示例输出$vc为undefined](https://zouyaoji.top/vue-cesium/#/zh-CN/component/quickstart)

请各位大神指点迷津

**Comment by zouyaoji** [comment]
啥版本啊？
